core/src/main/java/io/questdb/cairo/TableConverter.java

```diff
@@ -97,6 +97,8 @@ public static ObjList<TableToken> convertTables(CairoConfiguration configuration
                                     }
                                     txWriter.ofRW(path.trimTo(rootLen).concat(dirName).concat(TXN_FILE_NAME).$(), PartitionBy.DAY);
                                     txWriter.resetStructureVersionUnsafe();
+                                    txWriter.resetLagValuesUnsafe();
+                                    txWriter.resetLagValuesUnsafe();
```

Comment:
duplicate line?
could remove probably both of them as it will be done in line 114 anyway?
maybe it is worth merging the interaction with tx file together and have it at one place outside of the if-else statement?

Reply:
ah, yeah, typo